tests/test_E2_economics.py

```diff
@@ -203,7 +203,35 @@ def test_calculate_annual_dispatch_expenditures_pd_Series():
 
 def test_calculate_annual_dispatch_expenditures_else():
     with pytest.raises(TypeError):
-        E2.calculate_dispatch_expenditures([1, 2], flow, asset)
+        E2.calculate_dispatch_expenditures("str", flow, asset)
+
+
+def test_calculate_annual_dispatch_expenditures_list_scalars():
+    dispatch_expenditure = E2.calculate_dispatch_expenditures(
+        dispatch_price=[1, 1], flow=flow, asset=asset
+    )
+    assert dispatch_expenditure == 6
+
+
+def test_calculate_annual_dispatch_expenditures_list_pd_series():
+    dispatch_expenditure = E2.calculate_dispatch_expenditures(
+        dispatch_price=[1, pd.Series([1, 2, 3])], flow=flow, asset=asset
+    )
+    assert dispatch_expenditure == 9
+
+
+def test_calculate_annual_dispatch_expenditures_nested_list():
+    dispatch_expenditure = E2.calculate_dispatch_expenditures(
```

Comment:
some docstrings exaining what is being tested here and maybe assert tests would be nice :)

Reply:
I added some assert debug messages.